src/main/java/uk/gov/homeoffice/digital/sas/accruals/controllers/AccrualsController.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.homeoffice.digital.sas.accruals.controllers;
+
+import java.time.LocalDate;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.homeoffice.digital.sas.accruals.model.Accrual;
+import uk.gov.homeoffice.digital.sas.accruals.services.AccrualService;
+import uk.gov.homeoffice.digital.sas.jparest.web.ApiResponse;
+
+@Slf4j
+@RestController
+@RequestMapping("/resources/accruals")
```

Comment:
Looking for a difference URL here for the custom endpoints. 
`resources` is for the CRUD operations on a specific resource. The idea being that clients consuming the API have a well understood set of methods that can be called against any service using the REST API with custom endpoints being exposed on a difference prefix.
I don't know what to suggest here though. I've been referring to custom endpoints as business endpoints so it could be `/business/...` Also, I'm not 100% what is the endpoint we're exposing here.
Is it `/resources/accruals/getAccrualsImpactedByTimeEntry` because that's not very RESTful

Reply:
Reviewed with @taomoh @JWeston-HO  & @sewerynmi. My guess of the url was wrong and the url was the same as the existing GET url. The route is chosen by the unique parameters in the query string. We decided that the custom endpoint was RESTful not sufficiently different to require a different path.